lib/perl/Genome/Process.pm

```diff
@@ -355,7 +355,8 @@ sub lsf_job_id {
         if ($existing) {
             $existing->delete;
         }
-        $self->add_note(header_text => $self->lsf_job_id_header, body_text => $new_value);
+        my $n = $self->add_note(header_text => $self->lsf_job_id_header, body_text => $new_value);
+        $n->body_text($new_value); #ignore any system-generated sudo message for this note.
```

Comment:
Am I reading this incorrectly, or are we just assigning the same value twice in a row (once in `add` and once manually). What is causing the value to be different?

Reply:
This line in `create` is causing the value to be different:
https://github.com/genome/genome/blob/385fe08f29f6d8a93d57143b9f483b3345f6aa25/lib/perl/Genome/MiscNote.pm#L59
It's not correct IMO to be using a note to store this value... but here we are! So at least for now it'd be better if it gets stored correctly without the extra sudo message.

Reply:
Cool, makes sense! Would it be clearer/less confusing for the reader if we didn't assign the value initially and then just assigned it manually on the next line so there's only one "assignment?"

Reply:
I'm not convinced it would be.  The question then becomes "Why wasn't this assigned above?" instead of "Why is this assigned twice?" :smile:

Reply:
Hah, fair! Yeah, the real problem seems to be the constructor mutating a value you gave it without you asking it to do so - but I imagine changing that would have larger consequences.